### PR TITLE
Add metrics jcache

### PR DESCRIPTION
**Pull request by henri-tremblay**
This is a JCache module for metrics. It should works with any JCache compatible cache and was tested with Ehcache inside JHipster.

It was implemented by @anthonydahanne and I from Terracotta

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,11 +21,11 @@
         <module>metrics-core</module>
         <module>metrics-healthchecks</module>
         <module>metrics-ehcache</module>
-        <module>metrics-jcache</module>
         <module>metrics-ganglia</module>
         <module>metrics-graphite</module>
         <module>metrics-httpclient</module>
         <module>metrics-httpasyncclient</module>
+        <module>metrics-jcache</module>
```
Comment (anthonydahanne) [inline pom.xml]: hehe, not sure what this commit is bringing :-)

Comment (henri-tremblay) [inline pom.xml]: It is putting jcache at the right place. All modules are in alphabetical order

Comment (anthonydahanne) [inline pom.xml]: ah ! ok then ! good catch

**Comment by henri-tremblay** [comment]
Hi. I know the contributors of this project are probably overloaded but I would like a head's up.

I want to know if you are ok for adding a jcache module? When this PR can then be processed if a different matter.

**Comment by arteam** [comment]
@anthonydahanne and @henri-tremblay, 

Sorry to for the late response. I think this is a very fine module. The JCache specification has a JSR and supported by many caching libraries, so it makes sense to include and support it as one of the main modules. The implementation is rather concise, and I don't see downsides of merging it. Thank you and Terracotta for contributing it.

**Comment by henri-tremblay** [comment]
Awesome. Thanks

**Comment by henri-tremblay** [comment]
Hi. When do you think the next release will be performed?

**Comment by arteam** [comment]
Hi @henri-tremblay, I recently merged a bunch of pending pull requests to the 3.2 branch, and last week there were not so much activity on the project, so I think it's time to prepare a release. I will try to reach out to @ryantenney about it this week (he has the deploy rights to the `io.dropwizard.metrics` namespace in the Maven Central repository)

**Pull request merged**